tests/unit_tests/c_wrappers/res/config/test_config_content.py

```diff
@@ -74,9 +112,35 @@ def test_config_content_as_dict(tmpdir):
             ConfigKeys.NUM_REALIZATIONS: 42,
             ConfigKeys.QUEUE_OPTION: [["SLURM", "MAX_RUNNING", "50"]],
             "MULTIPLE_KEY_VALUE": [6, 24, 12],
-            ConfigKeys.DATA_FILE: [
-                str(Path.cwd() / existing_file_2),
-                str(Path.cwd() / existing_file_1),
-            ],
+            ConfigKeys.DATA_FILE: str(Path.cwd() / existing_file_2),
             ConfigKeys.REFCASE: str(Path.cwd() / existing_file_1),
         }
+
+
+def test_config_content_as_dict_single_value_keys(tmpdir):
+    with tmpdir.as_cwd():
```

Comment:
sorry, quick question: i can't seem to figure out where this `tmpdir` is coming from - it seems like a fixture to me, but i cannot find its definition.

a short comment: the pattern of having a tmpdir and chdiring into it, to do stuff, is implemented in the `use_tmpdir` fixture and can be used by just adding `@pytest.mark.usefixtures("use_tmpdir")`
of course then you drop the argument `tmpdir`, I believe, and have to instead access the current directory by using `os.getcwd`... so as i'm not sure what `tmpdir` does exactly here, i'll leave it at your discretion - you probably know about `use_tmpdir` already anyways...

Reply:
`tmpdir` is a pytest default fixture. 

And it mainly does exactly what `@pytest.mark.usefixtures("use_tmpdir")` does only that you do not have to call `os.getcwd()` to get the directory you are in. 

Also, I am not a big fun of implementing functionality that I can get for free from a package we already require for tests.

Reply:
that's why i couldn't find it ^^

ah, that makes a lot of sense! great, carry on : )